driver/src/main/java/org/kaazing/k3po/driver/internal/netty/channel/ChannelAddress.java

```diff
@@ -25,6 +26,13 @@
 public class ChannelAddress extends SocketAddress {
 
     private static final long serialVersionUID = 1L;
+    public static final Comparator<ChannelAddress> ADDRESS_COMPARATOR = new Comparator<ChannelAddress>() {
+        @Override
+        public int compare(ChannelAddress o1, ChannelAddress o2) {
+            return o1.getLocation().compareTo(o2.getLocation());
```

Comment:
transport comparison needs to be added here.

But now it doesn't work yet as bindings has:
http://localhost:8000/connections @ tcp://localhost:8000
candidate address :
http://localhost:8000/connections @ tcp://127.0.0.1:8000